tests/test_multisend.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from ape import networks
+
+from ape_safe import MultiSend
+from ape_safe.exceptions import SafeLogicError
+
+
+def test_asset(vault, token):
+    assert vault.asset() == token
+
+
+def test_default_operation(safe, token, vault):
+    with networks.ethereum.local.use_provider("foundry"):
+        ms = MultiSend()
+        ms.inject()
+        amount = token.balanceOf(safe)
+        ms.add(token.approve, vault, safe)
+        ms.add(vault.transfer, safe, amount)
+        receipt = ms(sender=safe)
+        assert receipt.txn_hash
+
+
+def test_no_operation(safe, token, vault):
+    with networks.ethereum.local.use_provider("foundry"):
```

Comment:
Since this is used a couple times, and to save an indent, we should make a fixture for it, like this:

```python
import pytest

@pytest.fixture
def foundry(networks):
  with networks.ethereum.local.use_provider("foundry") as provider:
    yield provider
```

This will automatically manage the network context so you can de-indent everything here.
You can put this in the `conftest.py` file.